src/Mailer/Message.php

```diff
@@ -347,23 +347,25 @@ public function getFrom(): array
     }
 
     /**
-     * Sets "sender" address.
+     * Sets the "sender" address (acting "on behalf of")
```

Comment:
```suggestion
     * Sets the "sender" address (sender acting "on behalf of" from)
```

Reply:
I don't know the cleanest way to write that.

Reply:
Which is why it's best to leave it short `Sets the "sender" address` and let users read the RFC to figure out what it does :slightly_smiling_face:

Reply:
agreed.

Reply:
> That's why it's best to leave it short `Sets the "sender" address` and let users read the RFC to figure out what it does slightly_smiling_face

I still don't agree. Now I might not need to check the code, but get redirected into reading a hightly technical RFC.

I don't expect the documentation of a framework code to educate me about a techical term but to tell me if a given function will do what I think it does (judging by its name and documentation).

By removing the ``"on behalf of"`` you successfully removed the last additional info (aparr from the mentioned technical RFC) this PR could have provided... :man_shrugging: